src/webgpu/api/validation/createPipelineLayout.spec.ts

```diff
@@ -162,3 +163,96 @@ g.test('bind_group_layouts,device_mismatch')
       t.device.createPipelineLayout({ bindGroupLayouts: [layout0, layout1] });
     }, mismatched);
   });
+
+const MaybeNullBindGroupLayoutTypes = ['Null', 'Undefined', 'Empty', 'NonEmpty'] as const;
```

Comment:
Combinatorially adding `undefined` increases the subcase count from 90 to 310, but I think that's a bit too many. This test is not that slow, but our combinatorial explosions in the CTS overall are quite out of hand so I'd like to be more cautious when we're adding new tests.

The null vs undefined thing is a trivial WebIDL check and we really just need one tiny test that makes sure null and undefined behave the same way.

That's a bit of extra work though. Would we get complete coverage by testing only the cases that have _exactly one_ `Null`/`Undefined`/`Empty`? This reduces the whole thing to just 30 subcases without having to write another test.

See #4107